WearOS/README.md

```diff
@@ -8,7 +8,7 @@ gradle-based Android Studio project that [supports ambient mode](https://develop
 Pre-requisites
 --------------
 
-- Android SDK v22
+- Android SDK v33
```

Comment:
Please verify that the sample still works with v33

Reply:
It should be 23 (the `minSdk `on the WearOS project.

Reply:
Hi @wangela. We should be having here the `compileSdk `version, the `minSdk `is not a [requirement to run the samples](https://github.com/googlemaps/android-samples/pull/1107#discussion_r1046893794). I am updating this to reflect the API 31.